### PR TITLE
Shared: Update shared dependencies

### PR DESCRIPTION
**Pull request by rihardsgravis**
# Description

Update shared dependencies:
- Add async `i18next` initialisation

## Type of change

- Enhancement (a non-breaking change which adds functionality)

# How Has This Been Tested?

Tested on:
- [x] macOS
- [x] iOS

# Checklist:

- [x] My code follows the style guidelines for this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] For changes to `shared`: If applicable, I have verified that my changes are implemented correctly in `desktop` and `mobile`

### CODE REVIEW

**Review by rajivshah3 — APPROVED** [review]
Tested on iOS

File: src/mobile/testConfig.js [hunk 1]
```diff
@@ -1,4 +1,6 @@
 import { configure } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
+import { i18nextInit } from 'shared-modules/libs/i18next';
 
 configure({ adapter: new Adapter() });
+i18nextInit().then(Promise.resolve());
```
Comment (laumair) [inline src/mobile/testConfig.js]: Is the `then` block necessary here?

**Review by laumair — APPROVED** [review]
Looks good to me!

File: src/mobile/testConfig.js (hunk 1 above)
Comment (rajivshah3) [reply src/mobile/testConfig.js]: I'll check. Probably not

Comment (rajivshah3) [reply src/mobile/testConfig.js]: Nope, it wasn't necessary. I'm not sure why I thought it was 🤔

**Pull request merged**